ssl-impl/src/test/java/io/crate/protocols/postgres/SslReqHandlerTest.java

```diff
@@ -120,8 +114,10 @@ public void testClassLoadingFallback() {
         }
     }
 
-    @Test(expected = SslConfigurationException.class)
+    @Test
     public void testClassLoadingWithInvalidConfiguration() {
+        expectedException.expect(SslConfigurationException.class);
+        expectedException.expectMessage("Failed to build SSL configuration");
```

Comment:
The exception message is allowed to change arbitrarily here. Important is only that we fail the test in case we have a type of `SslConfigurationException`.